setup.py

```diff
@@ -25,7 +25,7 @@
         "py-ecc==1.4.2",
         "rlp==0.4.7",
         "eth-keys==0.1.0b3",
-        "trie>=0.3.2",
+        "trie==0.3.2",
```

Comment:
What was the breakage with `1.0.0`?  I thought it was backwards compatible.

Reply:
I haven't dug into the root cause. The error message is like:
https://travis-ci.org/hwwhww/py-evm/jobs/324634985
```
tests/json-fixtures/test_state.py:247: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
evm/vm/base.py:39: in __init__
    self.block = block_class.from_header(header=header, chaindb=self.chaindb)
evm/vm/forks/frontier/blocks.py:94: in from_header
    transactions = chaindb.get_block_transactions(header, cls.get_transaction_class())
.tox/py35-native-state-byzantium/lib/python3.5/site-packages/eth_utils/functional.py:22: in inner
    return callback(fn(*args, **kwargs))
evm/db/chain.py:163: in get_block_transactions
    if transaction_key in transaction_db:
.tox/py35-native-state-byzantium/lib/python3.5/site-packages/trie/hexary.py:397: in __contains__
    return self.exists(key)
.tox/py35-native-state-byzantium/lib/python3.5/site-packages/trie/hexary.py:106: in exists
    return self.get(key) != BLANK_NODE
.tox/py35-native-state-byzantium/lib/python3.5/site-packages/trie/hexary.py:62: in get
    root_node = self._get_node(self.root_hash)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <trie.Trie object at 0x7f6c2cc46908>, node_hash = None
    def _get_node(self, node_hash):
        if node_hash == BLANK_NODE:
            return BLANK_NODE
        elif node_hash == BLANK_NODE_HASH:
            return BLANK_NODE
    
>       if len(node_hash) < 32:
E       TypeError: object of type 'NoneType' has no len()
```

Reply:
That's because `Trie.__init__()` doesn't upcall -- it just issues a warning

Reply:
fixed in #249